packages/fetch-http-handler/src/fetch-http-handler.ts

```diff
@@ -127,7 +127,7 @@ export class FetchHttpHandler implements HttpHandler<FetchHttpHandlerConfig> {
       requestOptions.keepalive = keepAlive;
     }
 
-    let removeSignalEventListener = null as null | (() => void);
+    let removeSignalEventListener = () => {};
```

Comment:
```suggestion
    // react-native promises cannot accept non-function as a then/catch/finally step.
    let removeSignalEventListener = () => {};
```

Reply:
I don't think the comment is necessary, as this fix changes `removeSignalEventListener` type to function?